applications/content-creator-applications/controller/applicationController.js

```diff
@@ -0,0 +1,69 @@
+const { ContentCreatorModel } = require('../../../models/ContentCreators');
+const { UserModel } = require('../../../models/Users');
+const mail = require('../../../helpers/email');
+
+const kindRegardsPT = 'Atenciosamente, Equipe Educado';
```

Comment:
Is the project being extended to more languages? Otherwise this seem unnecessary, small detail so its whatever.
If we are are extending to more languages we should think about what we want that system to look like